src/software/backend/radio_backend.h

```diff
@@ -29,10 +29,4 @@ class RadioBackend : public Backend
 
     // The interface that lets us send primitives to the robots over radio
     RadioOutput radio_output;
-
-    std::optional<World> most_recently_received_world;
```

Comment:
I'm assuming this is from the branch you're based on, not your intended changes?

Reply:
yup put it back

I think its unused though, we can probably safely remote it at some point